gdx/src/com/badlogic/gdx/maps/objects/PointMapObject.java

```diff
@@ -0,0 +1,29 @@
+package com.badlogic.gdx.maps.objects;
```

Comment:
could you please add licence header, you can find it in any other libgdx java file.

Reply:
PR updated